### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by jshaughn**
- for consistency and simplicity, migrate from html anchors to markdown image syntax
- add some top and bottom margin to img styling so that we can remove a bunch of inline `<br />`
- update remaining `</br>` to recommended `<br />`

...and a couple more change:

- update Releases to currently be master and pre-1.47.
- remove remaining .adoc and update README
-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+```
+make serve
+```
+
+{{% alert title="Podman" color="warning" %}}
+If you are using podman, you need to set `CONTAINER_RUNTIME` when starting the server, as such:
+```
+CONTAINER_RUNTIME=podman make serve
+```
+{{% /alert %}}
```
Comment (jmazzitelli) [inline README.md]:
```suggestion
{{% alert title="Docker" color="warning" %}}
If you are using docker, you need to set `DORP` when starting the server, as such:
\```
make -e DORP=docker make serve
\```
{{% /alert %}}
```

**Review by jmazzitelli — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jmazzitelli) [reply README.md]: Ignore the backslashes in my suggestion above - I can't get github to show the diff if I use triple-ticks - it sees it as the end of the suggestion when it really is part of the change.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+```
+make serve
+```
+
+{{% alert title="Docker" color="warning" %}}
+If you are using docker, you need to set `DORP` when starting the server, as such:
+```
+make -e DORP=docker make serve
+```
+{{% alert %}}
```
Comment (jmazzitelli) [inline README.md]:
```suggestion
> :warning: 
> If you are using Docker, you need to set `DORP` when starting the server; e.g:
> ```
> make -e DORP=docker make serve
> ```
```

**Review by jmazzitelli — CHANGES_REQUESTED** [review]
This is one way you can do a warning. Not sure if there is a more recommended alert box in markdown.

**Review by jmazzitelli — APPROVED** [review]
lgtm

**Review by israel-hdez — COMMENTED** [review]
In general I see this OK. Comments:

* Probably, the `scripts/build-archive.sh` can disappear. Most likely that branching dancing will need to be done inside Jenkins :-/
* We need to find a place for the Netlify badge or text to comply with their Open Source policy (read third bullet in https://www.netlify.com/legal/open-source-policy).
* Those new `img` styles are breaking the blog/videos cards in the landing page :-(

**Comment by israel-hdez** [comment]
@jshaughn I'll merge and I'll fix the styles in my #429 PR.

**Pull request merged**